### PR TITLE
STP-3754: Update component versioning

### PR DESCRIPTION
**Pull request by jld-dehn-hpe**
## Summary and Scope

SAT 2.5 will be available as part of a continuous release so that it can be used with the CSM 1.4.4 patch. The component versions in the 2.5 release notes need to be updated to reflect what is current before release.

## Issues and Related PRs

Resolves [STP-3754](https://jira-pro.it.hpe.com:8443/browse/STP-3754)

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: docs/release_notes/sat_2.5_release_notes.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Changes in SAT 2.5
 
-The 2.5.17 version of the SAT product includes:
+The 2.5.21 version of the SAT product includes:
```
Comment (haasken-hpe) [inline docs/release_notes/sat_2.5_release_notes.md]: I just created 2.5.22, actually. See PR here: https://github.com/Cray-HPE/sat-product-stream/pull/68/files

```suggestion
The 2.5.22 version of the SAT product includes:
```

File: docs/release_notes/sat_2.5_release_notes.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-- Version 3.21.4 of the `sat` python package and CLI.
+- Version 3.21.8 of the `sat` python package and CLI.
```
Comment (haasken-hpe) [inline docs/release_notes/sat_2.5_release_notes.md]: 2.5.22 bumps this once more:

```suggestion
- Version 3.21.9 of the `sat` python package and CLI.
```

**Review by haasken-hpe — APPROVED** [review]
Looks good. Note that I just created 2.5.22, so I added suggestions to update accordingly.

File: docs/release_notes/sat_2.5_release_notes.md (hunk 1 above)
Comment (jld-dehn-hpe) [reply docs/release_notes/sat_2.5_release_notes.md]: Updated in next commit.

File: docs/release_notes/sat_2.5_release_notes.md (hunk 2 above)
Comment (jld-dehn-hpe) [reply docs/release_notes/sat_2.5_release_notes.md]: Updated in next commit.

**Pull request merged**